index.js

```diff
@@ -86,14 +86,14 @@ export default function polyfill(Component) {
 
     Component.prototype.componentDidUpdate = function componentDidUpdatePolyfill(
       prevProps,
-      prevState
+      prevState,
+      maybeSnapshot
```

Comment:
This parameter will be `prevContext` for older 15.x versions of React, but since we check `this. __reactInternalSnapshot` first it should be okay.